schema/defs.yaml

```diff
@@ -74,6 +74,9 @@ description: Common schema definitions shared by all themes
           type: string
         dataset:
           type: string
+        recordId:
```

Comment:
This approach would result in a lot of duplication. For example, in Admin dataset for M5 TOM is providing most of the data and one name, ESRI providing bunch of names. To populate the sources we will need to repeat the ESRI provider id dozens of times.

Another approach would be to create a separate property to list provider name to provider id mappings once.

Reply:
That's a good point Vara. I'd be willing to just ship what we have to see how it goes down, and then if (as is perhaps likely) we feel things are way over-duplicated, we could move to a system with more "indirection" like you're suggesting.